docs/guides/04-data-driven-visualizations-part-1.md

```diff
@@ -0,0 +1,184 @@
+
+There are multiple classifying methods (quantiles, equal intervals...) and the classification can be applied to two different samples:
+- The entire dataset. `global*` classifiers will apply the classification to all source data. Ignoring filters or the presence of each feature in the viewport.
+- Viewport data. `viewport*` classifiers will apply the classification only to the features that are on the viewport. This includes filtering by the `filter:` styling property and filtering by checking that the feature is within the region covered by the screen at each moment. Changes in the view (map center/map zoom) will trigger an automatic re-computation of the classification.
+
+On top on that, you can also classify the data by a fixed list of breakpoints with the [`buckets()`](https://carto.com/developers/carto-vl/reference/#cartoexpressionsbuckets) function. For example, the expression `buckets($price, [10, 200])` will classify the features by its price into 3 different categories (buckets): the features that have a price less than 10, the features that have a price between 10 and 200, and the features that have a price higher than 200. It's important to note that there is always one more category than breakpoint. The `buckets` function can also be used with categorical inputs, we'll see that on the [next section](#One_to_one_mapping._One_category_-_one_color.).
+
+Let's see some maps with those. Do you see how `viewport*` classifiers are dynamic and changes in the map bounds change the result?
+
+<div class="example-map" style="margin: 20px auto !important">
+    <iframe
+        id="population-density-classified"
+        src="/developers/carto-vl/examples/maps/guides/ramp/population-density-classified.html"
+        width="100%"
+        height="1000"
+        frameBorder="0">
+    </iframe>
+    <a href="/developers/carto-vl/examples#example-population-density---classification">View my source code!</a>
+</div>
+
+##### A note about `filter:`
+
+`filter:` is a special styling property. Apart from multiplying the feature's color alpha channel by its value, it is used semantically to filter the dataset, which affects the `viewport*` classifiers and `viewport*` aggregators. When a feature's `filter:` value is above `0.5` we consider that the feature pass the filter, and the feature will be taken into account. When the value is below `0.5`, the feature is ignored (treated as non-existent) in all `viewport*` functions.
+
+#### Categorical properties
+
+Of course, not all data is numeric. Sometimes, it's just one value of a fixed number of possible values. For example, in an election map, we only have a fixed number of political parties. And in each region, only one party can win. This kind of data is what we call *categorical data*.
```

Comment:
It would be nice to enumerate all the ##### 5th level topics that are discussed in this epigraph:
- Encodings
- One to one mapping
- Others
- ...

Reply:
:+1: